ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/system-control-plane.j2

```diff
@@ -0,0 +1,44 @@
+{% if system.control_plane is arista.avd.defined %}
+
+## System Control-Plane
+
```

Comment:
To avoid duplicate empty lines in various combinations, this empty line should be enclosed in the `if` below.

Reply:
Done